app/View/Components/Tables/Table-carpool-drivers-edit.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\View\Components\Tables;
+
+use Closure;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class Table-carpool-drivers-edit extends Component
+{
+    /**
+     * Create a new component instance.
+     */
+    public function __construct()
```

Comment:
Add parameter 'public $carpoolDrivers' to constructor

Reply:
to what file I'm I adding this?

Reply:
Actually it's okay i have understood